index.html

```diff
@@ -17,59 +17,120 @@
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work3">
+                    <img src="img/work_img/Portfolio2.png" alt="work1 image">
+                    <h3 class="work-name">Multi-Post Stories</h3>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="work-skills grid-container">
+                        <li class="gird">CSS</li>
+                        <li class="gird">HTML</li>
+                        <li class="gird">Bootstrap</li>
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work4">
+                    <img src="img/work_img/Portfolio3.png" alt="work1 image">
+                    <h3 class="work-name">Multi-Post Stories</h3>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="work-skills grid-container">
+                        <li class="gird">CSS</li>
+                        <li class="gird">HTML</li>
+                        <li class="gird">Bootstrap</li>
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
             </div>
         </section>
-        <hr class="break-skill">
-       <section class="contact">
-           <ul class="contact-container">
-                <li>
-                    <a href="#"><img src="img/github.png" alt="github" class="github"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/linkin.png" alt="linkin" class="linkin"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/gesture.png" alt="gesture" class="gesture"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/twitter.png" alt="twitter" class="twitter"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/m.png" alt="m" class="m"></a>
-                </li>
-           </ul>
+        <section class="about-section">
+            <h2>About Me</h2>
+            <hr class="break-about">
+            <p class="description"> I am good at the following web technologies and skills. If you have any questions about web development, don't hesitate to contact me through the links below</p>
+            <div class="skill skill-container">
```

Comment:
- [ ] As the previous reviewer suggested. The section with the class `.about-section` should have a background image. You are only using the image in the skill container div. Kindly fix the issue.